base/shell/explorer/lang/zh-CN.rc

```diff
@@ -3,7 +3,7 @@
  * LICENSE:    GPL - See COPYING in the top level directory
  * FILE:       base/shell/explorer/lang/zh-CN.rc
  * PURPOSE:    Simplified Chinese Language File for ReactOS Explorer-New
- * TRANSLATOR: Song Fuchang <sfc_0@yahoo.com.cn>
+ * TRANSLATOR: 2017/12/06 njlyf2011 <njlyf2011@hotmail.com>
```

Comment:
Don't overwrite previous author, add a new line for yourself.

And please use your real name.

Reply:
Thank you for your correction.